specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -6925,6 +6925,101 @@
+        },
+        "type": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "subtype": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "id": {
+          "type": "string",
+          "description": "This field can be used as a passthrough for custom metadata about the matched text(s). The value of this field will appear with every match of its entity in the skill output."
+        },
+        "caseSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Defaults to false. Boolean value denoting whether comparisons with the entity name should be sensitive to character casing. Sample case insensitive matches of \"Microsoft\" could be: microsoft, microSoft, MICROSOFT."
+        },
+        "accentSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Defaults to false. Boolean value denoting whether comparisons with the entity name should be sensitive to accent."
+        },
+        "fuzzyEditDistance": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "description": "Defaults to 0. Maximum value of 5. Denotes the acceptable number of divergent characters that would still constitute a match with the entity name. The smallest possible fuzziness for any given match is returned. For instance, if the edit distance is set to 3, \"Windows10\" would still match \"Windows\", \"Windows10\" and \"Windows 7\". When case sensitivity is set to false, case differences do NOT count towards fuzziness tolerance, but otherwise do."
+        },
+        "defaultCaseSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Changes the default case sensitivity value for this entity. It be used to change the default value of all aliases caseSensitive values."
+        },
+        "defaultAccentSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "Changes the default accent sensitivity value for this entity. It be used to change the default value of all aliases accentSensitive values."
+        },
+        "defaultFuzzyEditDistance": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "description": "Changes the default fuzzy edit distance value for this entity. It can be used to change the default value of all aliases fuzzyEditDistance values."
+        },
+        "aliases": {
+          "type": "array",
+          "items": {
```

Comment:
This can be empty? If not, add `"minLength": 1`.

Reply:
If it can't be empty, will you not serialize it across the wire? If you will and it can be `null`, be sure to mark this field `x-nullable` as well.